src/js/tracks/text-track.js

```diff
@@ -156,15 +156,15 @@ class TextTrack extends Track {
     tt.cues_ = [];
     tt.activeCues_ = [];
 
-    let cues = new TextTrackCueList(tt.cues_);
-    let activeCues = new TextTrackCueList(tt.activeCues_);
+    const cues = new TextTrackCueList(tt.cues_);
+    const activeCues = new TextTrackCueList(tt.activeCues_);
     let changed = false;
-    let timeupdateHandler = Fn.bind(tt, function() {
+    const timeupdateHandler = Fn.bind(tt, function() {
 
       // Accessing this.activeCues for the side-effects of updating itself
       // due to it's nature as a getter function. Do not remove or cues will
       // stop updating!
-      this.activeCues;
+      this.activeCues; // eslint-disable-line
```

Comment:
you may want to wrap this in the specific rule that you are turning off. This will protect use  in case we change linting in the future and also make it more apparent why this line needs to be disabled (although the comment above does help already)